app/assets/stylesheets/pdf.css

```diff
@@ -1,51 +1,80 @@
-@import url("https://fonts.googleapis.com/css?family=Fira+Sans");
+.page-break {
```

Comment:
is this having an effect now?

Reply:
No, unfortunately it never seemed to work. If you have any ideas, I would love to hear them.

Reply:
nothing off the top - I can have a play with it - I'd say for now we should remove it otherwise its confusing